Owncloud iOs Client/Login/Login/LoginViewController.m

```diff
@@ -1661,6 +1664,23 @@ -(void) isConnectionToTheServerByUrlInOtherThread {
     [mCheckAccessToServer isConnectionToTheServerByUrl:[self getUrlToCheck]];
 }
 
+-(NSString *)stripIndexPhpOrAppsFilesFromUrl:(NSString *)url {
+    NSString *lastPath = k_remove_to_suffix;
```

Comment:
Do not create the variables because we have the constants.
If you have any problem cast to NSString (NSString *) to let know to the compiler that it is an NSString